mreg/migrations/0007_auto_20200905_1410.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.1 on 2020-09-05 12:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mreg', '0006_delete_modelchangelog'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='history',
+            name='data',
+            field=models.JSONField(),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='first_name',
+            field=models.CharField(blank=True, max_length=150, verbose_name='first name'),
+        ),
```

Comment:
The user migration comes from https://docs.djangoproject.com/en/3.1/releases/3.1/#abstractuser-first-name-max-length-increased-to-150